tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/rest/RestClientSpanDisabledTest.java

```diff
@@ -57,47 +61,58 @@
 import jakarta.ws.rs.core.Application;
 import jakarta.ws.rs.core.Response;
 
-@ExtendWith(ArquillianExtension.class)
-class RestClientSpanDisabledTest {
+class RestClientSpanDisabledTest extends Arquillian {
     @Deployment
     public static WebArchive createDeployment() {
         return ShrinkWrap.create(WebArchive.class)
-                .addAsResource(new StringAsset("client/mp-rest/url=${baseUri}\n"
-                        + "otel.experimental.sdk.enabled=false"),
+                .addClasses(InMemorySpanExporter.class, InMemorySpanExporterProvider.class)
+                .addAsLibrary(TestLibraries.AWAITILITY_LIB)
+                .addAsServiceProvider(ConfigurableSpanExporterProvider.class, InMemorySpanExporterProvider.class)
+                .addAsResource(new StringAsset("otel.experimental.sdk.enabled=true\notel.traces.exporter=in-memory"),
```

Comment:
This test is supposed to test the condition when `otel.experimental.sdk.enabled` is set to `false` so no trace should be created.

Reply:
Yep, that's a mistake and is repeated in all the Disabled and Default tests.

This test should have failed when I ran it locally. I suspect there is a delay in the spans being reported and so tests which expect zero spans pass immediately, even if the implementation does create spans. I think we should change the assertion when zero spans are expected to ensure we wait for a minimum amount of time to allow any spans which are created to be seen.

Reply:
[06f43c5](https://github.com/eclipse/microprofile-telemetry/pull/61/commits/06f43c5002d31bb8427e277d24df91efde34c9a6) made the default and disabled tests fail as expected. 
[9cc6cee](https://github.com/eclipse/microprofile-telemetry/pull/61/commits/9cc6cee80b2be13309374cfdbecb6e0fd2f1052b) makes the test pass again.